.changeset/ninety-queens-develop.md

```diff
@@ -0,0 +1,15 @@
+---
+'@strapi/icons': minor
+---
+
+**Feature :**
+Added 3 new icons for "xls", "zip", and "csv" from Phosphor icon library.
+
+**Changes :**
+- Added .svg files for the following icons:
+  - FileXls (for xls files)
+  - FileZip (for zip files)
+  - FileCsv (for csv files)
+
+**Reason for Inclusion :**
+These icons are needed for upcoming improvements in the Media Library.
```

Comment:
Actually this doesn't pass the lint test, could you update it to be simple single line text? basically just the name of the PR

Reply:
Sure, on it

Reply:
@remidej I have update the changesheet message.

Reply:
@remidej Yes you were correct. The error was due to missing newline only. Now it is resolved.